File: Programmers/Lv.0/주사위 게임 2/181930.java
class Solution {
    public int solution(int a, int b, int c) {
        int answer = 0;
        
        if (a != b && b != c) {
            answer = a + b + c;
        } else if (a != b || b != c) {
            answer = (a + b + c) * (a * a + b * b + c * c);
        } else {
            answer = (a * 3) * ((int) Math.pow(a, 2) * 3) * ((int) Math.pow(a, 3) * 3);
        }
        
        return answer;
    }
}

File: Programmers/Lv.0/주사위 게임 2/181930.js
function solution(a, b, c) {
    let answer = 0;
    
    if (a !== b && b != c && c != a) {
        answer = a + b + c;
    } else if (a !== b || b !== c) {
        answer = (a + b + c) * (a**2 + b**2 + c**2);
    } else {
        answer = (a * 3) * (a**2 * 3) * (a**3 * 3);
    }
    
    return answer;
}
